test/job_hooks_test.rb

```diff
@@ -342,6 +342,7 @@ def self.perform(history)
     Resque.dequeue(BeforeDequeueJob, history)
     @worker.work(0)
     assert_equal history, [:before_dequeue], "before_dequeue was not run"
+    assert_equal [:before_dequeue], history, "before_dequeue was not run"
```

Comment:
So I guess my original comment should maybe have been "Is the first `assert_equal` still needed?"